packages/sign_in_with_apple/example/android/app/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/example/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.aboutyou.dart_packages.sign_in_with_apple.example
```

Comment:
I like this bundle id more. Let's remove the other files and change the example app to use this one?

Reply:
Can also be a quick follow up to get this merged